parse_csv.py

```diff
@@ -0,0 +1,74 @@
+import csv
+import sys
+
+def strip_prefix(input_string, prefix):
+    if input_string.startswith(prefix):
+        return input_string[len(prefix):]
+    return input_string
+
+def add_to_output(export_output, row, prefix):
+    if row['URL'].startswith(prefix):
+        endurl = strip_prefix(row['URL'], prefix)
+        if endurl in export_output:
+            old_row = export_output[endurl]
+            for col in row.keys():
+                if col == "URL":
+                    row[col] = "https://www.parse.ly" + endurl
+                    continue
+                try:
+                    if old_row[col] != "":
+                        old_val = float(old_row[col])
+                    else:
+                        old_val = 0.0
+                    if row[col] != "":
+                        new_val = float(row[col])
+                    else:
+                        new_val = 0.0
+                    row[col] = str( old_val + new_val )
+                except ValueError:
+                    if row[col] == "":
+                        row[col] = old_row[col]
+                    elif old_row[col] == "":
```

Comment:
ah, this is odd - if the previous value was blank you'd just do nothing for the rest of the file traverse